pe/orchestration.py

```diff
@@ -101,9 +101,14 @@ def get_sub_dicts_for_exam(self, page_size: int = 50) -> List[Dict[str, Any]]:
                     next_params = page_info
                     page_num += 1
 
-        LOGGER.info(f'Gathered {len(sub_dicts)} submissions from Canvas')
-        LOGGER.debug(sub_dicts)
-        return sub_dicts
+        sub_dicts_with_scores: List[Dict[str, Any]] = list(filter((lambda x: x['score'] is not None), sub_dicts))
```

Comment:
Could the score be empty string as well?

Reply:
I'll look into this, but I don't think so.

Reply:
@zqian, I don't think an empty string score is possible. Canvas describes the field as "The raw score", and I think I've only ever seen floats as values here. Thus it won't be a string. When I remove the score completely, it comes back as a `null`.

Reply:
Great! Then it is non-issue.